vmm/src/config.rs

```diff
@@ -407,11 +407,26 @@ impl Default for RngConfig {
 pub struct FsConfig {
     pub tag: String,
     pub sock: PathBuf,
+    #[serde(default = "default_fsconfig_num_queues")]
     pub num_queues: usize,
+    #[serde(default = "default_fsconfig_queue_size")]
     pub queue_size: u16,
+    #[serde(default = "default_fsconfig_cache_size")]
     pub cache_size: Option<u64>,
 }
 
+fn default_fsconfig_num_queues() -> usize {
+    1
+}
+
+fn default_fsconfig_queue_size() -> u16 {
+    1024
+}
+
+fn default_fsconfig_cache_size() -> Option<u64> {
+    Some(0x0002_0000_0000)
+}
+
```

Comment:
The default values need to be documented through the `cloud-hypervisor.yaml` file.

Reply:
Oh so that's what the default value is meant for in the YAML file, it's just about documentation! I realized the default was not used/applied by the openAPI, which is why I removed it in the `NetConfig`. Ok I'll document the default values :)

Reply:
Done!